x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -500,6 +515,119 @@ export class AlertsClient {
     }
   }
 
+  public async getAlertSummary({
+    gte,
+    lte,
+    featureIds,
+    filter,
+    fixedInterval = '1m',
+    index,
+  }: GetAlertSummaryParams) {
+    try {
+      let indexToUse = index;
+      if (featureIds && !isEmpty(featureIds)) {
+        indexToUse = (
+          (await this.getAuthorizedAlertsIndices(
+            Array.isArray(featureIds) ? featureIds : [featureIds]
+          )) ?? []
+        ).join(',');
+      }
+
+      // first search for the alert by id, then use the alert info to check if user has access to it
+      const [activeResponse, recoveredResponse] = await Promise.all([
+        this.singleSearchAfterAndAudit({
+          index: indexToUse,
+          operation: ReadOperations.Get,
+          aggs: {
+            active_alerts_bucket: {
+              date_histogram: {
+                field: ALERT_TIME_RANGE,
+                fixed_interval: fixedInterval,
+                hard_bounds: {
+                  min: gte,
+                  max: lte,
+                },
+                extended_bounds: {
+                  min: gte,
+                  max: lte,
+                },
+              },
+            },
+          },
+          query: {
```

Comment:
Are we able to move the filter to the aggregation so this could be combined into a single request with two aggregations?